app/eth2wrap/eth2wrap_gen.go

```diff
@@ -39,6 +40,7 @@ type Client interface {
 	eth2client.BeaconBlockProposalProvider
 	eth2client.BeaconBlockSubmitter
 	eth2client.BeaconCommitteesProvider
+	eth2exp.BeaconCommitteeSubscriptionsSubmitter
```

Comment:
this isn't going to work since we need to support v1 AND v2 and adding `	eth2client.BeaconCommitteeSubscriptionsSubmitter` doesn't compile, suggest renaming to `eth2exp.BeaconCommitteeSubscriptionsSubmitterV2`

Reply:
done